src/config/dynamic-theme-fixes.config

```diff
@@ -14356,7 +14356,33 @@ CSS
 scribd.com
 
 INVERT
-.logo
+.wrapper__megamenu_top_bar .logo_button
+.scribd_blue_long_logo, .scribd_white_logo
+.auto__upload_index .icon_row .reason_cell .cell_icon
+.auto__static_about .about_content .fact, .auto__static_about .about_content .fun_facts
+.e584-3.x-image
```

Comment:
Try to avoid these kind of selectors, they are automatically generated and have a low guarantee that they won't be changed.

Reply:
What do you mean by "This type of selector" ? What kind should I be looking to use?

Reply:
The types that relies on random generated selectors. Try to see if this selector can be selected via a parent element that doesn't use a random selector.